src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -242,6 +242,19 @@ class ngraph::runtime::interpreter::INTExecutable : public Executable
     {
         const Node& node = *node_wrapper.get_node();
 
+        size_t opset_version = node.get_opset_version();
```

Comment:
In current version is used `op_version` instead of `opset_version` (`v0`, `v1` etc.) to determine if current backend supports the op.

 Can such version be merged to master? What is the status of @diyessi proposition? (https://github.com/NervanaSystems/ngraph/pull/3533/files)